packages/phpbb3.php

```diff
@@ -501,10 +503,14 @@ protected function ExportUserNotes() {
             'log_data' => array(
                 'Column' => 'Body',
                 'Type' => 'text',
-                'Filter' => function ($value) {
-                    $value = @unserialize($value);
+                'Filter' => function ($value, $field, $row) use (&$corruptedRecords) {
+                    $unserializedValue = @unserialize($value);
 
-                    return array_pop($value);
+                    if (!$unserializedValue) {
```

Comment:
I'm pretty sure an empty array is something that is supposed to fail.

How about:
```php
if (empty($unserializedValue) || !is_array($unserializedValue)) {
```